doc/workspaceConfiguration.md

```diff
@@ -0,0 +1,44 @@
+# Workspace configuration
+
+Some features of the webapp can be configured in your workspaces description. These are usually versioned in
```

Comment:
not sure about plural for _workspaces_ :slightly_smiling_face:

Reply:
Me neither, I'll remove it :smile: Thank you !